internal/validator/registry/registry.go

```diff
@@ -20,7 +20,7 @@ type ValidatorInitializer interface {
 }
 
 // ValidatorFactory is a function that creates a validator instance.
-type ValidatorFactory func(pass *codegen.Pass, field *ast.Field, expressions map[string]string, structName string, ruleName string) validator.Validator
+type ValidatorFactory func(pass *codegen.Pass, field *ast.Field, expressions map[string]string, structName, ruleName, parentPath string) validator.Validator
```

Comment:
Same remark than here

https://github.com/sivchari/govalid/pull/134#discussion_r2264624173

There is too many fields, using a struct would be better